plugins/inputs/pf/README.md

```diff
@@ -0,0 +1,65 @@
+# PF Plugin
+
+The pf plugin gathers counters from the FreeBSD/OpenBSD pf firewall.
```

Comment:
Looking through the code I don't see where we are parsing the counters, it looks like we are just collecting the state table?

Reply:
more precisely, number of entries and counters from the state table. will update.